where-to-travel/src/main/webapp/script.js

```diff
@@ -244,11 +244,11 @@ function addPlacesFromDirection(lat, lng, place_candidates) {
     let acceptablePlaces = [];
     let i = 0;
     for (i; i < listPlaces.size(); i++) {
-      const lat = listPlaces[i % 25].geometry.location.lat();
-      const lng = listPlaces[i % 25].geometry.location.lng();
+      const lat = listPlaces[i].geometry.location.lat();
+      const lng = listPlaces[i].geometry.location.lng();
       const destination = new google.maps.LatLng(lat, lng);
       userDestinations.push(destination);
-      if (i == 24 || i == listPlaces.size() - 1) {
+      if (i == 24 || i == listPlaces.size()-1) {
         let service = new google.maps.DistanceMatrixService();
         service.getDistanceMatrix({
           origins: [userLocation],
```

Comment:
Instead of creating a new userLocation object I think just doing [home] also works.

Reply:
Right